kits/arms/ar_control_sm.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+// HEBI C++ API files:
+#include "arm/arm.hpp"
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "group.hpp"
+#include "color.hpp"
+#include "util/vector_utils.h"
+#include <cmath>
+#include <Eigen/Dense>
+#include <thread>
+#include <chrono>
+
+// Common includes
+#include <iostream>
+
+using namespace hebi;
+
+enum class ArmControlState { STARTUP, HOMING, TELEOP, DISCONNECTED, EXIT };
+
+// This struct holds the inputs from the mobile IO device for controlling the arm
+struct ArmMobileIOInputs {
+  hebi::Vector3f phone_pos;
+  Eigen::Matrix3d phone_rot;
+  double ar_scaling{ 1.0f };
+  bool lock_toggle{ false };
+  bool locked{ true };
+  bool gripper_closed{ false };
+  bool home{ false };
+
+  // Optional constructor if you want to set values on creation
+  ArmMobileIOInputs(
+    const hebi::Vector3f& pos = hebi::Vector3f(0.0f, 0.0f, 0.0f),
+    const Eigen::Matrix3d& rot = Eigen::Matrix3d::Identity(),
+    const double scaling = 1.0f,
```

Comment:
Passing by value, the `const` doesn't really do anything here, and so can be dropped.  (Technically it prevents you from changing the local copy within the function, but this isn't usually the issue.  In the pass-by-value you're already decoupled from the calling function's context so you won't change the value there, which is what the `const` on a `const&` argument is trying to protect from.)

This is done a couple other times in this file, too...